### ISSUE TITLE
Link opens a new window without warning the user

### ISSUE THREAD

**Issue by dmytro-fedkiv-la** [dmytro-fedkiv-organization/level-ci-github-integration#30]
**Rule description**  
One or more A elements that have a target attribute set to the text value '_blank', without an ARIA-assigned role, not intentionally hidden in the DOM and available to assistive technologies, does not have a calculated accessible name or description value that includes a warning to the user that a new window will open i.e. 'new window'.

**Tool name**  
Access Engine

**Severity**  
High

**WCAG Success criteria**  
[3.2.5 Change on Request - Level AAA](https://www.w3.org/WAI/WCAG22/Understanding/change-on-request)

**Why it matters**  
Users who are blind, have low vision, or have mobility impairment all rely heavily on proper control of keyboard and reading focus when browsing web based content. When a new window opens without communicating this fact, users may struggle to return to the location from which the window was launched.

**What to do**  
Provide a mechanism to communicate that a new window opens, such as by using the <code>aria-label</code> attribute or <code>title</code> attribute to the link that includes the phrase such as "opens new window" or by using off-screen positioned CSS text or an icon, image, or SVG accessible name communicating the link opens a new window. Note, depending on the approach used you will want to make sure the phrase "opens new window" does not suppress the accessible name of the link from being communicated. <p>Note: The <code>title</code> attribute acts as an accessible description when a link contains an accessible name via link text, image with alternative in the anchor, etc. The description may only be announced when focused with tab by a screen reader and not when the arrow keys are used.</p> 

Alternatively links or controls that open new windows can be changed to not open in a new window. 

<p><em>Note-</em>WCAG 2.0 Level A conformance only requires not opening the window without user initiated action. Level AAA conformance requires an additional statement warning the user that the window will open on user action.</p>

**Task in the Level Access Platform**  
https://fig-860.hub.dev.dev.levelaccess.io/my-tasks/6ab41e8a57ba8831a725f5a5/6ab41f85d75f6371511e0478

**Comment by dmytro-fedkiv-la** [comment]
<a href="https://fig-860.hub.dev.dev.levelaccess.io/tasks/6ab41f85d75f6371511e0478/attachments/6ab41f85d75f6371511e0476/download" target="_blank" rel="noopener noreferrer">Linked Findings.csv</a>

**Comment by dmytro-fedkiv-la** [comment]
Task pushed to GitHub Issues by Dmytro Fedkiv from Level Access Platform.

Posted by Dmytro Fedkiv via Level Access
